README.md

```diff
@@ -1,2 +1,76 @@
-# manifest
-The feeless order book exchange that supercharges your on-chain trading abilities
+# Manifest
+
+The feeless order book exchange that supercharges your on-chain trading
+abilities Manifest is the next generation of orderbooks on Solana. It is
+optimized for freedom. No more permissioned markets. No more trading fees. No
+more large 2 SOL rent to start a market. Capital efficiency built in.
+
+## Comparison
+
+
+|  |    Openbook    | Phoenix  |Manifest              |
+|--|----------------|-------------------|----------------------|
+| Crank |Yes|No |No |
+| Fees |Yes |Yes |No|
+| Predefined lot sizes |Yes |Yes |No|
+| Anchor |Yes |Yes |No|
+| Rent|2 SOL |2 SOL |.002 SOL|
+| License|GPL |Business |GPL|
+| OpenOrders separation for read locking| Yes | No | Yes |
+| Swap accounts| 10 | 8 | 7 |
+| CU | | |
+|Instruction size | | | |
+|Silent failures | Yes| Yes| No|
+|Token 22 | No| No| Yes|
```

Comment:
Add wrapper architecture No, No, Yes?

Reply:
You could write a wrapper for any, just not built for it